app/controllers/api/v0/reactions_controller.rb

```diff
@@ -36,7 +42,7 @@ def valid_user
         user
       end
 
-      def organization_article?(reaction)
+      def org_article?(reaction)
         reaction.reactable_type == "Article" && reaction.reactable.organization_id
```

Comment:
Should we update this to check for the actual organization rather than just the id in the event that the organization is removed?

Reply:
Don't know if we want to do it in the same PR but yes!

Reply:
Holding you too that so I dont see it in HB in the future 😉